pages/getting-started/classes/travel-101.md

```diff
@@ -45,7 +45,12 @@ have access to OLU to complete the required IT security trainings._
 
 Once you are [set up in Concur]({% page "/first-time-travel-get-in-concur/" %})
 and your [travel card]({% page "/first-time-travel-travel-card/" %}) is on its
-way, there are three main parts of the travel process to be aware of:
+way, there are three main parts of the travel process to be aware of. First, travel
+is planned and booked in Concur and approved there. Timing will vary, but you
```

Comment:
Suggest taking it out of the passive voice. "First, you will book your travel and submit a plan to be approved in Concur. Your supervisor and the TTS travel team will review and approve"